js/borrower-speak.test.js

```diff
@@ -0,0 +1,12 @@
+const borrow = require('./borrower-speak');
+
+
+describe('borrower-speak test first test', () => {
```

Comment:
The final test name will be:

> borrower-speak test first test change string to lower case without specials

I believe the `describe`'s block name is a bit too long 😄 Try to fiddle with it to make the final test case more human friendly and correct